### PR TITLE
EA Periodic table widget 

### LINKED ISSUE #29394: Periodic table in EA
We want to transfer a trimmed down version of the current EA GUI into a periodic table tab. This will let users select the elements they want to inspect.

It should add the lines to the plot

### PR DESCRIPTION
**Pull request by tolu28-coder**
**Description of work.**
This pull  request adds a periodic table tab for the new Elemental analysis gui.

**To test:**
1. Before starting Mantid go to https://github.com/mantidproject/mantid/blob/master/Framework/Properties/Mantid.properties.template#L24 , add Muon/Elemental_Analysis_2.py to the end of the line and build Mantid.
2. On the periodic table clicking on any element should print information on lines selected which is the element symbol, transition , x value of line. and colour of line used.
3.  By unchecking the checkbox should spawn 2 buttons one to select the new data file and another to load the data, and by rechecking the checkbox the default datafile should be used. 

Fixes #29394.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Muon/GUI/ElementalAnalysis2/periodic_table_tab/periodic_table_tab_presenter.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```
Comment (AnthonyLim23) [inline scripts/Muon/GUI/ElementalAnalysis2/periodic_table_tab/periodic_table_tab_presenter.py:1]: are these just copies of the original?

Comment (tolu28-coder) [reply scripts/Muon/GUI/ElementalAnalysis2/periodic_table_tab/periodic_table_tab_presenter.py:1]: Alot of the code was copied from https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py
but some new feautures were added

Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis2/periodic_table_tab/periodic_table_tab_presenter.py:1]: you should inherit the code instead and add new functionality in the derived class

**Comment by AnthonyLim23** [comment]
There is no one to work on this, so I am closing it.

**Pull request closed without merging**